src/components/Slider/SliderInfix.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { bem, INFIX_KIND } from "./SliderCommons";
+import { useSliderInfix, useSliderSelection } from "./SliderContext";
+import Label from "../Label/Label";
+import Icon from "../Icon/Icon";
+
+const defaultIconProps = {
+  clickable: false,
+  ignoreFocusStyle: true
+};
+
+function getCurrentLabel({ isPostfix, isRange, valueText }) {
+  if (!isRange) {
+    return valueText;
+  }
+  if (isPostfix) {
+    return valueText[1];
+  }
+  return valueText[0];
+}
+
+function useSliderInfixComponent(kind) {
+  const { prefix, postfix, indicateSelection } = useSliderInfix();
```

Comment:
please move the hooks to a separate file

Reply:
You prefer to move it to SliderHooks or at separate SliderInfixHooks?

Reply:
all the hooks can be in one SliderHooks file

Reply:
if the hooks are only for a specific component - IMO they could be in a file SliderInfixHooks